modules/sdk-core/src/bitgo/keychain/keychains.ts

```diff
@@ -305,4 +312,110 @@ export class Keychains implements IKeychains {
       backupProvider: params.backupProvider,
     });
   }
+
+  /**
+   * It parses the JSON downloaded from the OVC for platform (BitGo),
+   * and creates a corresponding TSS BitGo key. It also returns the JSON that needs
+   * to be uploaded back to the OVCs containing the BitGo -> OVC shares.
+   * @param ovcOutputJson JSON format of the file downloaded from the OVC for platform
+   * @returns {BitGoKeyFromOvcShares}
+   */
+  async createTssBitGoKeyFromOvcShares(ovcOutputJson: OvcToBitGoJson): Promise<BitgoKeyFromOvcShares> {
```

Comment:
```suggestion
  async createTssBitGoKeyFromOvcShares(ovcOutputJson: unknown): Promise<BitgoKeyFromOvcShares> {
```

Since we are using io-ts, we can rely on the decoded value to have a type.